community/examples/intel/README.md

```diff
@@ -378,6 +377,7 @@ See the [File System](https://docs.daos.io/v2.0/user/filesystem/) section of the
 The container will need to by unmounted before you log out.  If this is not done it can leave open file handles and prevent the container from being mounted when you log in again.
 
 ```bash
+popd
```

Comment:
Is this intentional? I don't see a mention in commit and I lack the relevant expertise.